Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Restarted from build #2, stage run nic case
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (clone)
Stage "clone" skipped due to this build restarting at stage "run nic case"
[8mha:////4BMellQCKYcT15C6njtpBMRMPcx1v+Ml2BYg9lj5HcLFAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaWGsbAmNJ0AWEZb8zwLrbuWJvJp3kLiJlZNMMm+a93rDOic4UbLcG+wdZu14DKOti0+U+lugiXu6ck2YKRguzSSpM+cFJRUDS1gDKwEbgzpQdmgLbIVXD9UGhba9lFS/o4DGdQM8gYlqLiqVL8wJdvexy4Q/z18BzDEWYIfa/QeSFDiovAAAAA==[0m[Pipeline] }
[8mha:////4OXOK0zaKdhyxUbs/zGHojhUfmLEUhqJHWICM+k606WNAAAAox+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQRbmZgYK1tC4wmQRYQl/7PAult5Iq/mHSRuYuUkk8yb5r3esM4JTpQs9wZ7h1k7HsNo6+ITpf4WaOKerlwTZgqGSzNJ6sx5QUnFwBLWwErAxqAOlB3aAlvh1UO1QaFtLyXV7yigcd0AT2CimotK5Qtzgt197DLhz/NXAHOMBdihdv8B+dJhs7wAAAA=[0m[Pipeline] // stage
[8mha:////4HFZTA94r5T36LLcCEmmPWVO3p/trW1XEP0v6cHlwFbjAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0YUGIiTXKwglCEkLa6P82SWknTsTVuAOBSkxYsmRbst7zBXWKcKToeGux85i0530YXUl8othdA028pQvXhImC5dJOkow9LVVStrCIVbASsLaoAyWPLsNGtOqumqDQNeccy3YQUHkzwAOYKOSsYv6WOcL2NppE+OP8BcDcZ2C74v3nV78BTzecD8AAAAA=[0m[Pipeline] stage
[8mha:////4PW1R6wrRuLDMfXXQCg2rohPfCKrNDnrLqJEB0zcG6/NAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQDJUJUtJYbXmBiY5xYd8G+kFS8iK/xBwKRqNhqZ6XVPF9Q5gQHSl42DtuAuQ6yi72fmhwotZdIg2zoLGvCTNFJ7QZN1h1n1MQO5ogCFgqWDutIOaBnWKnG3E0VDfrqxGna9gqKYG/wAKEmM5vEXxgTrK+9zYQ/z18BjB2D2DEU283nWL4Bsam+msEAAAA=[0m[Pipeline] { (set args)
Stage "set args" skipped due to this build restarting at stage "run nic case"
[8mha:////4A2AToW0H2anYE8m90a6j2EeXpcs3JZx8OcXh4vd/J2AAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOJCakRFa6XhBSYxxo5159gOTsWL+Bp/IBCJii1WO9vM8wXrGODIQaNV1BuKrUHvRj0vzBz6q+OMli/YMkV2ChuVG+7UacGGk4IlRQkrARtFreNoSCfYCivvsnKSdHVOYf4OAkrTDfCAQszmJEP6whRgdxu7yPTz/BXA5H2Csq4/tX8DHHS23b4AAAA=[0m[Pipeline] }
[8mha:////4JzFhAG4/s02BQKK5iTl/LC8g8NOw10asO74FotINtcOAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRdAhR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Qb3eCbP8BWAQ01r0AAAA=[0m[Pipeline] // stage
[8mha:////4KrJzHXeAN4Zy3GWv8GYfZim4qRT8KCk+B7FcZ64hEjBAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2Gw/x/INGJa+18EAAAA=[0m[Pipeline] stage
[8mha:////4A6A9O1vyYtnm686LUJxskpjau4zm9zFrdnzgk/5DUGhAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUCDbE1DXKwglCE0La6P/yk9JOnIircQdaKjHhwbItWe/1hiIynIidaCy2HmPtRRd6NyUxELfXQINo6CJqwkjBCmUHRcZWS1WULCzKclhJWFusA0WPLsFGNvqhy6DRlefE03aUkHtzhydkciInzelbRobtrTeR8Mf5C4CxS5Dv9rMd5mfxAbNWnxXCAAAA[0m[Pipeline] { (run nic case)
[8mha:////4F69FYvmoe7TYwtgh0tvicuYBtQZbS9M8h4xL1d3u5QvAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiJKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHN8W7Eii+AAAA[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics -s -m nicSriov2hours --html=/root/testconfig/testlogs/HtmlReport/nic_sriov2hours_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 16 items / 15 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-02-10 15:46:23 - NicTestFixture.py - [line:28] - root - INFO - Starting to run test_PI_Networking_SRIOV_StressTrafficBetweenPFVFAndTestClientFor2Hours_VM_Guest[uvm1.img-uvm2.img]
2022-02-10 15:46:23 - NicTestFixture.py - [line:29] - root - INFO - setup log for testing in the fixture class fbefore
2022-02-10 15:46:23 - test_network_sriov_stress.py - [line:31] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-02-10 15:46:23 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-02-10 15:46:23 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-02-10 15:46:23 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.62
2022-02-10 15:46:23 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-02-10 15:46:23 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-02-10 15:46:23 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-02-10 15:46:23 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.60
2022-02-10 15:46:26 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -s | grep ens2f0 | awk '{print $1;}'
2022-02-10 15:46:26 - sshInstance.py - [line:48] - root - INFO - 0000:4b:00.0

2022-02-10 15:46:31 - test_network_sriov_stress.py - [line:87] - root - INFO - pf_id was found: 0000:4b:00.0

2022-02-10 15:46:31 - test_network_sriov_stress.py - [line:489] - root - INFO - format_pf_id is 0000\:4b\:00.0

2022-02-10 15:46:31 - test_network_sriov_stress.py - [line:492] - root - INFO - pf_id_key: 4b
2022-02-10 15:46:37 - sshInstance.py - [line:92] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu Feb 10 15:40:15 2022 from 10.112.97.60
[root@WW25ICX2 ~]# ifconfig ens2f0 192.168.10.12
[root@WW25ICX2 ~]# 
2022-02-10 15:46:40 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens2f0
2022-02-10 15:46:43 - sshInstance.py - [line:44] - root - INFO - echo 4 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-02-10 15:46:43 - sshInstance.py - [line:48] - root - INFO - 
2022-02-10 15:46:48 - test_network_sriov_stress.py - [line:101] - root - INFO - Successfully Virtualized PF to VFs.
2022-02-10 15:46:51 - sshInstance.py - [line:44] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-02-10 15:46:52 - sshInstance.py - [line:48] - root - INFO - 4b:01.0
4b:01.1
4b:01.2
4b:01.3

2022-02-10 15:46:57 - test_network_sriov_stress.py - [line:115] - root - INFO - Succssfully Gotten vf_id list:

2022-02-10 15:47:00 - sshInstance.py - [line:44] - root - INFO -  modinfo vfio-pci | grep "vfio_pci" | awk '{ print $2;}'
2022-02-10 15:47:00 - sshInstance.py - [line:48] - root - INFO - vfio_pci

2022-02-10 15:47:05 - test_network_sriov_stress.py - [line:168] - root - INFO - Successfuly modeprobe vfio-pci
2022-02-10 15:47:08 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.0
2022-02-10 15:47:08 - sshInstance.py - [line:48] - root - INFO - 
2022-02-10 15:47:13 - test_network_sriov_stress.py - [line:147] - root - INFO - Successfuly bound VF to vfio-pci
2022-02-10 15:47:16 - sshInstance.py - [line:44] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.1
2022-02-10 15:47:16 - sshInstance.py - [line:48] - root - INFO - 
2022-02-10 15:47:21 - test_network_sriov_stress.py - [line:147] - root - INFO - Successfuly bound VF to vfio-pci
2022-02-10 15:47:21 - test_network_sriov_stress.py - [line:515] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-02-10 15:47:24 - sshInstance.py - [line:44] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-02-10 15:47:24 - sshInstance.py - [line:48] - root - INFO - 0000:4b:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-02-10 15:47:29 - test_network_sriov_stress.py - [line:526] - root - INFO - Successfully get vf device id: 8086:1889
2022-02-10 15:47:32 - sshInstance.py - [line:44] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-02-10 15:47:32 - sshInstance.py - [line:48] - root - INFO - no

2022-02-10 15:47:37 - test_network_sriov_stress.py - [line:177] - root - INFO - Default network is active? no

2022-02-10 15:47:37 - test_network_sriov_stress.py - [line:186] - root - INFO - virsh net-start default
2022-02-10 15:47:40 - sshInstance.py - [line:44] - root - INFO - virsh net-start default
2022-02-10 15:47:41 - sshInstance.py - [line:48] - root - INFO - Network default started


2022-02-10 15:47:46 - test_network_sriov_stress.py - [line:190] - root - INFO - Successfully started network default.
2022-02-10 15:47:50 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noaautoconsole

** (process:8935): [1;33mWARNING[0m **: [34m15:56:42.895[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-02-10 15:47:52 - sshInstance.py - [line:69] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-02-10 15:47:55 - test_network_sriov_stress.py - [line:212] - root - INFO - Successfully created a VM: VMtest1
2022-02-10 15:47:59 - sshInstance.py - [line:69] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:9005): [1;33mWARNING[0m **: [34m15:56:51.702[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-02-10 15:48:01 - sshInstance.py - [line:69] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-02-10 15:48:04 - test_network_sriov_stress.py - [line:212] - root - INFO - Successfully created a VM: VMtest2
2022-02-10 15:48:07 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-02-10 15:48:07 - sshInstance.py - [line:48] - root - INFO - 
2022-02-10 15:48:12 - test_network_sriov_stress.py - [line:229] - root - ERROR - not found IP of VMtest1
2022-02-10 15:49:15 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-02-10 15:49:15 - sshInstance.py - [line:48] - root - INFO - 192.168.122.172/24

2022-02-10 15:49:20 - test_network_sriov_stress.py - [line:224] - root - INFO - Successfully got VMtest1 IP: 192.168.122.172/24

2022-02-10 15:49:23 - sshInstance.py - [line:44] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-02-10 15:49:23 - sshInstance.py - [line:48] - root - INFO - 192.168.122.115/24

2022-02-10 15:49:28 - test_network_sriov_stress.py - [line:224] - root - INFO - Successfully got VMtest2 IP: 192.168.122.115/24

2022-02-10 15:49:32 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.172
PING 192.168.122.172 (192.168.122.172) 56(84) bytes of data.
64 bytes from 192.168.122.172: icmp_seq=1 ttl=64 time=0.284 ms
2022-02-10 15:49:33 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=2 ttl=64 time=0.478 ms
2022-02-10 15:49:34 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=3 ttl=64 time=0.312 ms
2022-02-10 15:49:35 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=4 ttl=64 time=0.331 ms
2022-02-10 15:49:36 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=5 ttl=64 time=0.469 ms

--- 192.168.122.172 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.284/0.374/0.478/0.085 ms
[root@WW25ICX2 ~]# 
2022-02-10 15:49:45 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.172
The authenticity of host '192.168.122.172 (192.168.122.172)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-02-10 15:49:54 - sshInstance.py - [line:92] - root - INFO - yes
Warning: Permanently added '192.168.122.172' (ECDSA) to the list of known hosts.
root@192.168.122.172's password: 
2022-02-10 15:50:03 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 02:58:53 AM EST
  System load:  0.0                Processes:             139
  Usage of /:   38.6% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.172
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 01:13:26 2022 from 192.168.122.1
root@localhost:~# 
2022-02-10 15:50:12 - sshInstance.py - [line:92] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-02-10 15:50:18 - test_network_sriov_stress.py - [line:322] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-02-10 15:50:24 - sshInstance.py - [line:92] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-02-10 15:50:29 - test_network_sriov_stress.py - [line:336] - root - INFO - Successfully got vf device name of vm1: ens5
2022-02-10 15:50:35 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-02-10 15:50:38 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-02-10 15:50:44 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-02-10 15:50:54 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-02-10 15:51:03 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-02-10 15:51:12 - sshInstance.py - [line:92] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-02-10 15:51:21 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-02-10 15:51:30 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.172 closed.
[root@WW25ICX2 ~]# 
2022-02-10 15:51:37 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.115
PING 192.168.122.115 (192.168.122.115) 56(84) bytes of data.
64 bytes from 192.168.122.115: icmp_seq=1 ttl=64 time=0.273 ms
2022-02-10 15:51:38 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=2 ttl=64 time=0.323 ms
2022-02-10 15:51:39 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=3 ttl=64 time=0.471 ms
2022-02-10 15:51:40 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=4 ttl=64 time=0.462 ms
2022-02-10 15:51:41 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.115: icmp_seq=5 ttl=64 time=0.317 ms

--- 192.168.122.115 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4083ms
rtt min/avg/max/mdev = 0.273/0.369/0.471/0.082 ms
[root@WW25ICX2 ~]# 
2022-02-10 15:51:50 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.115
The authenticity of host '192.168.122.115 (192.168.122.115)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-02-10 15:51:59 - sshInstance.py - [line:92] - root - INFO - yes
Warning: Permanently added '192.168.122.115' (ECDSA) to the list of known hosts.
root@192.168.122.115's password: 
2022-02-10 15:52:08 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 03:00:58 AM EST
  System load:  0.0                Processes:             142
  Usage of /:   38.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.115
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 01:15:22 2022 from 192.168.122.1
root@localhost:~# 
2022-02-10 15:52:17 - sshInstance.py - [line:92] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-02-10 15:52:23 - test_network_sriov_stress.py - [line:322] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-02-10 15:52:29 - sshInstance.py - [line:92] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-02-10 15:52:34 - test_network_sriov_stress.py - [line:336] - root - INFO - Successfully got vf device name of vm1: ens5
2022-02-10 15:52:40 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 192.168.10.14
root@localhost:~# 
2022-02-10 15:52:43 - test_network_sriov_stress.py - [line:283] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-02-10 15:52:49 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-02-10 15:52:59 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-02-10 15:53:08 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-02-10 15:53:17 - sshInstance.py - [line:92] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-02-10 15:53:26 - sshInstance.py - [line:92] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.14  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 342 (342.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-02-10 15:53:35 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.115 closed.
[root@WW25ICX2 ~]# 
2022-02-10 15:53:42 - sshInstance.py - [line:69] - root - INFO - ping -c 5 192.168.122.172
PING 192.168.122.172 (192.168.122.172) 56(84) bytes of data.
64 bytes from 192.168.122.172: icmp_seq=1 ttl=64 time=0.268 ms
2022-02-10 15:53:43 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=2 ttl=64 time=0.273 ms
2022-02-10 15:53:44 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=3 ttl=64 time=0.390 ms
2022-02-10 15:53:45 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=4 ttl=64 time=0.266 ms
2022-02-10 15:53:46 - sshInstance.py - [line:69] - root - INFO - 64 bytes from 192.168.122.172: icmp_seq=5 ttl=64 time=0.466 ms

--- 192.168.122.172 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4127ms
rtt min/avg/max/mdev = 0.266/0.332/0.466/0.083 ms
[root@WW25ICX2 ~]# 
2022-02-10 15:53:55 - sshInstance.py - [line:92] - root - INFO - ssh root@192.168.122.172
root@192.168.122.172's password: 
2022-02-10 15:54:04 - sshInstance.py - [line:92] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Feb 2022 03:02:54 AM EST
  System load:  0.04               Processes:             127
  Usage of /:   38.6% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.172
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.13

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Feb 10 02:58:54 2022 from 192.168.122.1
root@localhost:~# 
2022-02-10 15:54:13 - sshInstance.py - [line:92] - root - INFO - nohup ping -w 7200 192.168.10.12 > ping_sut1.txt &
[1] 1081
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:54:22 - sshInstance.py - [line:92] - root - INFO - nohup ping6 -I ens5 -w 7200 fe80::b696:91ff:fe9c:f7a4 >ping6_sut1.txt &
[2] 1084
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:54:31 - sshInstance.py - [line:92] - root - INFO - nohup ping -w 7200 192.168.10.14 > ping_vm2.txt &
[3] 1086
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:54:40 - sshInstance.py - [line:92] - root - INFO - nohup ping6 -I ens5 -w 7200 fe80::1034:56ff:fe78:90ab >ping6_vm2.txt &
[4] 1087
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:54:49 - sshInstance.py - [line:92] - root - INFO - nohup ping -w 7200 192.168.10.11 > ping_sut2.txt &
[5] 1088
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:54:58 - sshInstance.py - [line:92] - root - INFO - nohup ping6 -I ens5 -w 7200 fe80::b696:91ff:fea5:9b45 >ping6_sut2.txt &
[6] 1089
root@localhost:~# 
root@localhost:~# nohup: ignoring input and redirecting stderr to stdout

2022-02-10 15:55:01 - test_network_sriov_stress.py - [line:592] - root - INFO - ---ping end -----
2022-02-10 17:55:07 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping_sut1.txt
--- 192.168.10.12 ping statistics ---
7032 packets transmitted, 7032 received, 0% packet loss, time 7199593ms
rtt min/avg/max/mdev = 0.036/0.167/0.314/0.026 ms
[1]   Done                    nohup ping -w 7200 192.168.10.12 > ping_sut1.txt
[2]   Done                    nohup ping6 -I ens5 -w 7200 fe80::b696:91ff:fe9c:f7a4 > ping6_sut1.txt
[3]   Done                    nohup ping -w 7200 192.168.10.14 > ping_vm2.txt
[4]   Done                    nohup ping6 -I ens5 -w 7200 fe80::1034:56ff:fe78:90ab > ping6_vm2.txt
[5]-  Done                    nohup ping -w 7200 192.168.10.11 > ping_sut2.txt
[6]+  Done                    nohup ping6 -I ens5 -w 7200 fe80::b696:91ff:fea5:9b45 > ping6_sut2.txt
root@localhost:~# 
2022-02-10 17:55:16 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping6_sut1.txt
--- fe80::b696:91ff:fe9c:f7a4 ping statistics ---
7032 packets transmitted, 7032 received, 0% packet loss, time 7199738ms
rtt min/avg/max/mdev = 0.036/0.177/0.341/0.029 ms
root@localhost:~# 
2022-02-10 17:55:25 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping_vm2.txt
--- 192.168.10.14 ping statistics ---
7032 packets transmitted, 7032 received, 0% packet loss, time 7199752ms
rtt min/avg/max/mdev = 0.037/0.188/0.342/0.052 ms
root@localhost:~# 
2022-02-10 17:55:34 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping6_vm2.txt
--- fe80::1034:56ff:fe78:90ab ping statistics ---
7032 packets transmitted, 7032 received, 0% packet loss, time 7199158ms
rtt min/avg/max/mdev = 0.029/0.201/1.115/0.087 ms
root@localhost:~# 
2022-02-10 17:55:44 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping_sut2.txt
--- 192.168.10.11 ping statistics ---
7032 packets transmitted, 7032 received, 0% packet loss, time 7199749ms
rtt min/avg/max/mdev = 0.041/0.188/0.575/0.034 ms
root@localhost:~# 
2022-02-10 17:55:53 - sshInstance.py - [line:92] - root - INFO - tail -n 3 /root/ping6_sut2.txt
--- fe80::b696:91ff:fea5:9b45 ping statistics ---
7033 packets transmitted, 7033 received, 0% packet loss, time 7199954ms
rtt min/avg/max/mdev = 0.038/0.194/1.492/0.086 ms
root@localhost:~# 
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping sut1
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping6 sut1
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping vm2
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping6 vm2
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping sut2
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:641] - root - INFO - Successfully ping6 sut2
2022-02-10 17:55:56 - test_network_sriov_stress.py - [line:642] - root - INFO - check ---end---
2022-02-10 17:56:02 - sshInstance.py - [line:92] - root - INFO - exit
logout
Connection to 192.168.122.172 closed.
[root@WW25ICX2 ~]# 
.2022-02-10 17:56:05 - test_network_sriov_stress.py - [line:57] - root - INFO - this is the teardown of Test_Network_SRIOV_Stress.
2022-02-10 17:56:08 - sshInstance.py - [line:44] - root - INFO - virsh destroy VMtest2
2022-02-10 17:56:08 - sshInstance.py - [line:48] - root - INFO - Domain VMtest2 destroyed


2022-02-10 17:56:13 - test_network_sriov_stress.py - [line:258] - root - INFO - Successfully shut down the VM: VMtest2
2022-02-10 17:56:16 - sshInstance.py - [line:44] - root - INFO - virsh undefine VMtest2
2022-02-10 17:56:16 - sshInstance.py - [line:48] - root - INFO - Domain VMtest2 has been undefined


2022-02-10 17:56:21 - test_network_sriov_stress.py - [line:268] - root - INFO - Successfully removed the VM: VMtest2
2022-02-10 17:56:24 - sshInstance.py - [line:44] - root - INFO - virsh destroy VMtest1
2022-02-10 17:56:24 - sshInstance.py - [line:48] - root - INFO - Domain VMtest1 destroyed


2022-02-10 17:56:29 - test_network_sriov_stress.py - [line:258] - root - INFO - Successfully shut down the VM: VMtest1
2022-02-10 17:56:32 - sshInstance.py - [line:44] - root - INFO - virsh undefine VMtest1
2022-02-10 17:56:32 - sshInstance.py - [line:48] - root - INFO - Domain VMtest1 has been undefined


2022-02-10 17:56:37 - test_network_sriov_stress.py - [line:268] - root - INFO - Successfully removed the VM: VMtest1
2022-02-10 17:56:37 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-02-10 17:56:37 - test_network_sriov_stress.py - [line:63] - root - INFO - =============================================Finished run the test case===============================================


=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:37
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_iperf_client_4hour.py:37: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:70
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_iperf_client_4hour.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:40
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_mtu.py:40: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:79
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_mtu.py:79: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:362
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:362: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:478
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:478: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:776
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_network_sriov_stress.py:776: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:39
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_nics.py:39: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:64
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_nics.py:64: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:87
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_nics.py:87: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:115
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_nics.py:115: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:154
  /root/.jenkins/workspace/test-test-nic-sriov-stress-2hours/npxtester/Tests/testNics/test_nics.py:154: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov2hours_case_report.html -
========== 1 passed, 15 deselected, 12 warnings in 7814.80s (2:10:14) ==========
[8mha:////4MMryhz2Fh91KFywmrHlj1fO0tpQvW7FOKmGkyEX1j4KAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiJESDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQG9gN7HvgAAAA==[0m[Pipeline] }
[8mha:////4C1LddmHqeX8+n6bU05IuiYWnvzfd8dajZBpmUPj7VH9AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQG7xuaSvgAAAA==[0m[Pipeline] // stage
[8mha:////4BxcZ3Nz10TQFng3Vqu7PPYnMFJn0D62ZxRVC4ddEdnFAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREgUSoqK10vACkxjjxLoL9oWk4kV8jT8QEYmKrXammdcbyxRx5OhUY6n1lCqvutC76amBY3sNPKiGL6piShysKu1Qcm1PM5YsFvOyHAuNlaUqcPLkBGvdmIcpgiFXnCVO7qCR+/qOJzI9lcVE+cIYsbn1dWL6df4GMHadIN/uBdnuA0RgNeK9AAAA[0m[Pipeline] }
[8mha:////4P1iJTZqYp14fU49pdve+jYFyVCE6LRxpJWs5Yxga0YfAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRokKIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsPV9wxQb0AAAA=[0m[Pipeline] // node
[8mha:////4BkWRocResLnIaVc0hiQi+7q3bcBJ3jWwxCk6fpiel5ZAAAAoh+LCAAAAAAAAP9tjTESgjAQRT84FraWHiI4tI6VbYbGE0SIMZDZxWQRKk/k1byDjMxY+av/XvNeb6xTxJGjU62lzlOqverD4OanRo7dNfCoWr6omilxsKqyY8WNPS1YsVgsy3KsNDaW6sDJkxNsdWsepgiGXHGWOLuDRu6bO57I9FwWE+ULU8TuNjSJ6df5G8DU94K83Auy8gMe/LK9vQAAAA==[0m[Pipeline] End of Pipeline
Finished: SUCCESS
